pkg/util/conditions_compare_test.go

```diff
@@ -0,0 +1,423 @@
+/*
+Copyright 2023 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestCompareConditions(t *testing.T) {
+	tests := []struct {
+		name       string
+		conditions1 *kedav1alpha1.Conditions
+		conditions2 *kedav1alpha1.Conditions
+		expected   bool
```

Comment:
This section is not aligned and static checks are failing https://github.com/kedacore/keda/actions/runs/15509388525/job/43668304162?pr=6817